docs/_snippets/patient-response-no-facilities-or-links.mdx

```diff
@@ -7,6 +7,10 @@
   An external Patient ID to associate to a Patient in Metriport.
 </ResponseField>
 
+<ResponseField name="ehrIds" type="string">
+  A map the contains all the EHR external Patient IDs.
```

Comment:
let's be explicit on what this is, and not assume the user knows how our system works - how about:

```
   If the patient is created in Metriport through one of our [EHR apps](link) - this property will be populated
   with the ID from the EHR, and included within the webhook payloads.
```